src/main/java/uk/ac/ebi/ena/webin/cli/WebinCli.java

```diff
@@ -648,4 +665,18 @@ private static void checkVersion(boolean test) {
                 .map(dir -> getSafeOutputDir(dir))
                 .toArray(String[]::new);
     }
+
+    private enum EnaSubmissionTool {

```

Comment:
- Please change the EnaSubmissionTool name to WebinSubmissionTool (Webin-CLI and other Webin services are not used only for ENA submissions).
- Please add WEBIN_CLI as the other enumerated value.